WalletWasabi/BitcoinCore/Rpc/CachedRpcClient.cs

```diff
@@ -67,6 +68,16 @@ public override async Task<Block> GetBlockAsync(uint blockHeight, CancellationTo
 			() => base.GetBlockAsync(blockHeight, cancellationToken)).ConfigureAwait(false);
 	}
 
+	public override async Task<VerboseBlockInfo> GetVerboseBlockAsync(uint256 blockId, CancellationToken cancellationToken = default)
+	{
+		string cacheKey = $"{nameof(GetVerboseBlockAsync)}:{blockId}";
+
+		return await Cache.AtomicGetOrCreateAsync(
+			cacheKey,
+			CacheOptions(10, 4),
```

Comment:
The _verbose_ json blocks are much bigger than the raw binary blocks and then, if well we have never specified a size for memory cache on the server, it makes no sense to have the "virtual" size for both kind of encoded blocks. Imo this here should be at least the double.

Reply:
Not sure I understand what you're suggesting here. Maybe this?
```suggestion
			CacheOptions(5, 4),
```

Reply:
```suggestion
			CacheOptions(20, 4),
```

Reply:
Ah yeah, I misunderstood the size param.